lntest/harness.go

```diff
@@ -1443,25 +1442,14 @@ func (h *HarnessTest) fundCoins(amt btcutil.Amount, target *node.HarnessNode,
 		PkScript: addrScript,
 		Value:    int64(amt),
 	}
-	h.miner.SendOutput(output, defaultMinerFeeRate)
+	txid := h.miner.SendOutput(output, defaultMinerFeeRate)
```

Comment:
Nit: should we make sure this txid is never `nil`

Reply:
The call `SendOutput` is wrapped in an assertion already, and I think it's actually nice to have the test panicked.